content.json

```diff
@@ -72,7 +72,16 @@
       "description": "As someone who’s used to implementing asynchronous programming in JS, implementing it in Python came as a surprise. Here’s what I learned.",
       "date": "2022-08-22",
       "img": "/static/python-vs-javascript-dealing-with-the-quirks-of-async-await.jpg",
-      "tags": [],
+      "tags": [
```

Comment:
these are too many tags.. let's keep it to 4.. "async",
        "await",
        "python",
        "javascript",